src/card/aggregate-status/aggregate-status-card.html

```diff
@@ -41,7 +48,14 @@ <h2 class="card-pf-title">
       {{$ctrl.status.title}}
     </span>
   </h2>
-  <div class="card-pf-body">
+  <div class="card-pf-body" ng-class="{'show-spinner': $ctrl.showSpinner}">
```

Comment:
Try moving the `ng-style` to here, it would be consistent with the `!mini` HTML above where we added the `ng-style`